### DISCUSSION TITLE
client connections fail on separate networks - "Failed to connect to relay server: Please try later"

### DISCUSSION THREAD

**Discussion by ll3N1GmAll** [rustdesk/rustdesk#1134] [Q&A]
I am getting the following error when trying to connect to a client: "Failed to connect to relay server: Please try later"

1 client is on the same network as the relay server. This client can only access the relay server via the local private IP. External clients have the public IP configured to reach the relay server.

I have verified via packet captures that the external client is making it through the firewall and to the relay server successfully. Networking is not the problem. 

Is it possible that 2 clients (1 internal on the same subnet as the relay server and 1 external on the internet) with different IPs configured for the relay and ID servers are unable to communicate due to them having 2 different IPs configured for the relay and ID servers? When all clients are local and have the same local private IP configured, everything works. As soon as 1 client is on a different network and the relay server IP is updated to the public IP of the relay server, the connection always fails with the error: "Failed to connect to relay server: Please try later"

**Comment by rustdesk** [comment]
unsure if related https://github.com/rustdesk/rustdesk-server/issues/24

**Reply by ll3N1GmAll** [reply]
Sorry I missed this one. Thanks, yes! This is almost exactly my scenario.

**Comment by ll3N1GmAll** [comment]
Duplicate issue - https://github.com/rustdesk/rustdesk-server/issues/24
